builds/misc/templates/build-rocksdb.yaml

```diff
@@ -33,7 +33,12 @@ stages:
       steps:
       - script: |
           /usr/bin/newgrp docker <<EONG
-          scripts/linux/buildRocksDb.sh --output-dir $(Build.ArtifactStagingDirectory) --postfix $(postfix) --build-number $(Build.BuildNumber) --arch $(arch)
+          scripts/linux/buildRocksDb.sh \
+            --output-dir $(Build.ArtifactStagingDirectory) \
+            --postfix $(postfix) \
+            --build-number $(Build.BuildNumber) \
+            --arch $(arch) \
+            --source-map /etc/iotedge-source-map.json
```

Comment:
Nit: Please add `"` around the input arguments (especially `postfix`)

Reply:
They weren't there before, so I won't add them in this PR. I agree it's probably a good idea to quote variables unless you have a good reason not to, but in this case each argument's value is demonstrably benign (especially postfix, which is one of amd64, arm32v7, or arm64).